structlog_sentry/__init__.py

```diff
@@ -0,0 +1,39 @@
+import logging
+import os
+import sys
+
+from sentry_sdk import capture_exception, capture_message
+
+
+class SentryProcessor:
+    """Sentry processor for structlog. Uses Sentry SDK to capture events in Sentry."""
+
+    def __init__(self, sentry_log_level=logging.ERROR):
+        self.sentry_log_level = sentry_log_level
+
+    def _log(self, event_dict, level="error"):
+        exc_info = event_dict.pop("exc_info", sys.exc_info())
+        has_exc_info = exc_info and exc_info != (None, None, None)
+
+        if has_exc_info and logging.getLevelName(level.upper()) >= logging.ERROR:
```

Comment:
Exceptions can be logged with any level. Condition for level shouldn't be here.

Reply:
What if someone sets level to `CRITICAL` only? Should we log errors then?

Reply:
Condition to log critical only is covered in `do_log` on line 29. It will not call `_log()` for error level in that case.

Reply:
In other words `do_log` handles levels. Here we should care only about if we have `exc_info` and want to call `capture_exception` instead of `capture_message`.

Reply:
Yes, you are right, I missed that.